llvm/lib/Transforms/Instrumentation/TypeSanitizer.cpp

```diff
@@ -886,6 +960,11 @@ PreservedAnalyses TypeSanitizerPass::run(Module &M,
   for (Function &F : M) {
     const TargetLibraryInfo &TLI = FAM.getResult<TargetLibraryAnalysis>(F);
     TySan.sanitizeFunction(F, TLI);
+    if (ClVerifyOutlinedInstrumentation && ClOutlineInstrumentation) {
+      ClOutlineInstrumentation = false;
+      TySan.sanitizeFunction(F, TLI);
+      ClOutlineInstrumentation = true;
+    }
```

Comment:
Could you add a brief comment documenting the logic here. It verifies both the outline and inline instrumentation behaves the same, if there are no violations; the outlined instrumentation may still trigger a verification failure the inlined instrumentation would not and vice versa?